### ISSUE TITLE
Shelby County Schools - Alabama

### ISSUE THREAD

**Issue by nateflowers** [flowersjg/schoolcalendars#30]
The crawler found a ton of google calendars for Shelby County Schools in Alabama, but we appear to have added them correctly last year, because they are mostly showing up.

Also, we have a lot of schools in our listing that are maybe defunct? They are not listed on the Shelby County Schools site: http://www.shelbyed.k12.al.us/directory/school_listings.htm

Example: http://www.schoolcalendars.org/alabama/thompson-intermediate-school-8105304/

We don't have a google calendar for this one, and it's likely because they list no such school on their site.

In short: the big chunk of Shelby County Schools in Return of Calendar Names : Rows 173:221 - can probably be left alone for now. I would not incorporate anything from Return of Calendar Names, since we renamed all of the calendars last year. 

Shelby County's PDF calendars are in rows 523 and 524, and should be added in like normal.
